rotkehlchen/tests/api/test_makerdao_vaults.py

```diff
@@ -891,3 +894,48 @@ def test_query_vaults_usdc_strange(rotkehlchen_api_server, ethereum_accounts):
     details = assert_proper_response_with_result(response)
     expected_details = [vault_7538_details]
     assert_serialized_lists_equal(expected_details, details)
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_modules', [['makerdao_vaults']])
+@pytest.mark.parametrize('mocked_proxies', [{
+    TEST_ADDRESS_1: '0x689D4C2229717f877A644A0aAd742D67E5D0a2FB',
+}])
+@pytest.mark.parametrize('have_decoders', [True])
+def test_delete_vault_owner(rotkehlchen_api_server: 'APIServer'):
+    """Check that deleting owner of a makerdao vault doesnt break anything"""
```

Comment:
Is important to be descriptive with the comments since in the future we will come here and won't know what was broken

```suggestion
    """Check that deleting owner of a makerdao vault doesnt break balance queries"""
```